src/Illuminate/Auth/Passwords/DatabaseTokenRepository.php

```diff
@@ -157,7 +161,7 @@ public function delete(CanResetPasswordContract $user)
      */
     public function deleteExpired()
     {
-        $expiredAt = Carbon::now()->subSeconds($this->expires);
+        $expiredAt = app(Factory::class)->now()->subSeconds($this->expires);
 
         $this->getTable()->where('created_at', '<', $expiredAt)->delete();
```

Comment:
Auth component doesn't load `app()` which is under `Illuminate\Foundation`, nor does it require `illuminate/container`.